dev/devfile.yaml

```diff
@@ -25,21 +25,20 @@ components:
           targetPort: 9080
           protocol: http
 commands:
-  - id: copy
-    exec:
-      component: devruntime
-      commandLine: echo "in the copy command"  && echo "...moving liberty"
-                                               && mkdir -p /projects/target/liberty
-                                               && mv -n /opt/ol/wlp /projects/target/liberty
-      workingDir: /
   - id: build
     exec:  
       component: devruntime 
       commandLine: if [ -e /projects/.disable-bld-cmd ];
                    then
                        echo "found the disable file" && echo "devBuild command will not run" && exit 0;
                    else
-                       echo "will run the devBuild command" && mvn -Dmaven.repo.local=/mvn/repository package
+                       echo "will run the devBuild command" && mkdir -p /projects/target/liberty
+                                                            && if [ ! -d /projects/target/liberty/wlp ];
+                                                                 then echo "...moving liberty"; mv /opt/ol/wlp /projects/target/liberty; touch ./.liberty-mv;
+                                                               elif [[ -d /projects/target/liberty/wlp && ! -e /projects/.liberty-mv ]];
+                                                                 then echo "STACK WARNING - LIBERTY RUNTIME WAS LOADED FROM HOST";
```

Comment:
Warning msg would be confusing if we did a 2nd package after the 1st failed (but the mv had already been done).

Reply:
nm..you handled this..sorry